src/css/style.css

```diff
@@ -0,0 +1,188 @@
+/* Import Component Library */
+@import url("https://neog-component-library-arun.netlify.app/src/components/main.css");
+
+/* welcome page */
+#welcomepage .container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 20px;
+  align-items: center;
+  height: 100vh;
+}
+
+#welcomepage .container .left .hero {
+  max-width: 15ch;
+  margin: 0 auto;
+}
+
+#welcomepage .container .get-started {
+  margin-top: 7.5rem;
+}
+
+#welcome {
+  background-color: var(--white-cream);
+}
+
+.icons {
+  font-size: 1.5rem;
+}
+
+.hero-section {
+  background-color: var(--grey-light);
+}
+
+.hero-section-left {
+  align-items: center;
+}
+
+.hero-section-left .h1 {
+  max-width: 16ch;
+}
+
+.hero-section-left .sub-title {
+  max-width: 50ch;
+}
+
+#login-page main,
+#signup-page main {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  height: calc(100vh - 120px);
+  place-items: center;
+}
+
+.wallpaper {
+  grid-column: 1 / span 6;
+  background-color: #ebebff;
+}
+
+.form-container {
+  grid-column: 7 / span 6;
+  margin: 0 30px;
+}
+
+@media screen and (max-width: 850px) {
+  .wallpaper {
+    display: none;
+  }
+  .form-container {
+    grid-column: 1/12;
+  }
+}
+
+.product-list {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(17.5rem, 1fr));
+  grid-gap: 1.25rem;
+  width: 90%;
+}
+
+.categories {
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(17.5rem, 1fr));
+  grid-gap: 1.25rem;
+  width: 90%;
+}
+
+.category-content {
+  display: inline-flex;
+  padding: 20px;
+}
+
+.category-content:hover {
+  box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2),
+    0px 4px 5px 0px rgba(0, 0, 0, 0.14), 0px 1px 10px 0px rgba(0, 0, 0, 0.12);
+}
+
+.category-content-left {
+  height: 64px;
+  width: 64px;
+  margin-right: 40px;
+  background-color: var(---white);
+  transition: all 0.3s;
+  line-height: 64px;
+  background-color: var(--white);
+}
+
+.category-content-right {
+  align-items: start;
+  justify-content: center;
+  gap: 0.5rem;
+}
+
+/* .category-section .category-content {
+  margin-bottom: 24px;
+} */
+
+.category-section .content {
+  margin: 0 auto;
+}
+
+.hero-section {
+  height: calc(100vh - 100px - 100px);
```

Comment:
I am not sure, but why can't we directly use calc(100vh - 200px) here?

Reply:
100 for each footer and header, will be easy to modify if one of them needs to be adjusted